rsync_to_rdisc.py

```diff
@@ -129,28 +136,28 @@ def connect_to_remote_server(host_keys, servers, user, run_file):
         except OSError:
             if hpc_server == servers[-1]:
                 send_mail_lost_hpc(" and ".join(servers), run_file)
-                sys.exit("Connection to HPC transfernodes are lost.")
+                sys.exit("Connection to HPC transfer nodes are lost.")
         except (timeout, ssh_exception.SSHException, ssh_exception.AuthenticationException):
             if hpc_server == servers[-1]:
                 Path(run_file).unlink()
                 sys.exit("HPC connection timeout/SSHException/AuthenticationException")
     return client, hpc_server
 
 
-def get_folders_remote_server(client, folder_dic, run_file, transferred_set):
+def get_folders_remote_server(client, mount_transfer_settings, run_file, transferred_set):
```

Comment:
You only use `mount_transfer_settings['transfers']`. Instead you could provide transfers directly.